hook.php

```diff
@@ -231,6 +235,50 @@ function plugin_escalade_install() {
       $migration->migrationOneTable('glpi_plugin_escalade_configs');
    }
 
+   // ## update to 2.2.0
+   // add new fields
+   if (! FieldExists('glpi_plugin_escalade_configs', 'remove_delete_requester_user_btn')) {
+      $migration->addField('glpi_plugin_escalade_configs', 'remove_delete_requester_user_btn',
+                           'bool', ['value' => 1]);
+   }
+   if (! FieldExists('glpi_plugin_escalade_configs', 'remove_delete_requester_group_btn')) {
+      $migration->addField('glpi_plugin_escalade_configs', 'remove_delete_requester_group_btn',
+                           'bool', ['value' => 1]);
+   }
+   if (! FieldExists('glpi_plugin_escalade_configs', 'remove_delete_watcher_user_btn')) {
+      $migration->addField('glpi_plugin_escalade_configs', 'remove_delete_watcher_user_btn',
+                           'bool', ['value' => 1]);
+   }
+   if (! FieldExists('glpi_plugin_escalade_configs', 'remove_delete_watcher_group_btn')) {
+      $migration->addField('glpi_plugin_escalade_configs', 'remove_delete_watcher_group_btn',
+                           'bool', ['value' => 1]);
+   }
+
+   // migrate old fields
+   if (FieldExists('glpi_plugin_escalade_configs', 'remove_delete_user_btn')) {
```

Comment:
Sounds like the existing fields has not been changed in tables creation (same for the 2 foloowing fields).